read-pipeline.tldr

```diff
@@ -0,0 +1 @@
36623,"rotation":0,"isLocked":false,"opacity":0.5,"meta":{},"id":"shape:wDwJ6wmGlzcpM0K7aAaxR","type":"text","props":{"color":"light-blue","size":"s","w":325,"text":"42|1024|3|bl..1|128|bl..2|256|","font":"mono","align":"middle","autoSize":true,"scale":0.6226526843205429},"parentId":"shape:p6bNP3PmALLhnnmW8U0YV","index":"a1","typeName":"shape"},{"x":43.9165702433,"y":22.41755575123352,"rotation":0,"isLocked":false,"opacity":0.5,"meta":{},"id":"shape:xAc3rKW7CBIOnaKzFTtU9","type":"text","props":{"color":"light-blue","size":"s","w":227.8046875,"text":"bafy...stuff/bag..two","font":"mono","align":"middle","autoSize":true,"scale":0.614470935824795},"parentId":"shape:JE1Q24dRjWijxpJlRngRZ","index":"a2","typeName":"shape"},{"x":159.79201631044236,"y":64.15755405582536,"rotation":0,"isLocked":false,"opacity":1,"meta":{},"id":"shape:z34IvKTo65SIIo-vTPbfm","type":"arrow","parentId":"shape:3t38pqnPshl6eaQ8zSh1t","index":"aG","props":{"dash":"solid","size":"s","fill":"solid","color":"light-blue","labelColor":"black","bend":0,"start":{"type":"point","x":1.7007889841840642,"y":1.9346474695093554},"end":{"type":"point","x":-9.62959678777314,"y":14.830525611046488},"arrowheadStart":"none","arrowheadEnd":"arrow","text":"","labelPosition":0.5,"font":"mono"},"typeName":"shape"},{"x":-61.381112648288166,"y":-4.677397450418312,"rotation":0,"isLocked":false,"opacity":1,"meta":{},"id":"shape:zQf9NnhBkVnbMo2e7BL3e","type":"group","parentId":"shape:M4G_SsrsWBlP6Q2lky4AD","index":"a1","props":{},"typeName":"shape"},{"x":-59.281852898423466,"y":-13.351931210703356,"rotation":0,"isLocked":false,"opacity":1,"meta":{},"id":"shape:zsukfoCj0tUKmZhKo22rU","type":"group","parentId":"shape:X0v5sMZoEmAmni6MZMch0","index":"a1","props":{},"typeName":"shape"},{"followingUserId":null,"opacityForNextShape":1,"stylesForNextShape":{},"brush":null,"scribbles":[],"cursor":{"type":"default","rotation":0},"isFocusMode":false,"exportBackground":true,"isDebugMode":false,"isToolLocked":false,"screenBounds":{"x":0,"y":0,"w":1213,"h":852},"insets":[false,false,false,false],"zoomBrush":null,"isGridMode":false,"isPenMode":false,"chatMessage":"","isChatting":false,"highlightedUserIds":[],"canMoveCamera":true,"isFocused":true,"devicePixelRatio":2,"isCoarsePointer":false,"isHoveringCanvas":false,"openMenus":["main menu","main-menu-sub.file"],"isChangingStyle":false,"isReadonly":false,"meta":{},"duplicateProps":null,"id":"instance:instance","currentPageId":"page:page","typeName":"instance"},{"x":660.3196135010137,"y":783.1571105809057,"z":0.47983117693813965,"meta":{},"id":"camera:page:page","typeName":"camera"},{"editingShapeId":null,"croppingShapeId":null,"selectedShapeIds":[],"hoveredShapeId":null,"erasingShapeIds":[],"hintingShapeIds":[],"focusedGroupId":null,"meta":{},"id":"instance_page_state:page:page","pageId":"page:page","typeName":"instance_page_state"},{"id":"pointer:pointer","typeName":"pointer","x":-590.1858969619184,"y":-696.4566948941541,"lastActivityTimestamp":1714667796351,"meta":{}}]}
```

Comment:
can you also add svg so that is visible here? Also I think we should add this to the docs folder

Reply:
we have all the other diagrams in the infra, I have no idea at the moment what this diagram is, but I wonder if we should just have a repo for all the diagrams instead

Reply:
I did not intend to publish it here